Description: Correct pod NAME issue in LedgerSMB/GL.pm
 Is missing a pod NAME entry altogether.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472600&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/GL.pm
+++ b/LedgerSMB/GL.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB:GL - General Ledger backend code
+
+=cut
+
 #=====================================================================
 # LedgerSMB
 # Small Medium Business Accounting software
